docs/DeveloperGuide.md

```diff
@@ -780,16 +786,28 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a stock
+### Deleting stocks
 
-1. Deleting a stock while all relevant stock are shown.
+1. Deleting stocks from a given list.
 
    1. Prerequisites: List all stocks by default or use the `find` command. Multiple stocks in the list.
 
    1. Test case: `delete sn/1111111`<br>
-      Expected: Stock of the serial number 1111111 is deleted from the inventory. 
+      Expected: Stock with the serial number 1111111 is deleted from the inventory. 
       Details of the deleted stock shown in the status message.
-
+      
+   1. Test case: `delete sn/1111111 sn/11111111`<br>
+      Expected: Stock with the serial number 1111111 is deleted from the inventory. 
+      Duplicate serial number is ignored. Details of the deleted stock shown in the status message.
+      
+   1. Test case: `delete sn/1111111 sn/22222222`<br>
+      Expected: Both stocks with the serial numbers 1111111 and 22222222 are deleted from the inventory. 
+      Details of the deleted stock shown in the status message.
+      
+   1. Test case: `delete sn/1111111 sn/33333333` (no stock has the serial number `33333333`) <br>
+        Expected: Only the existing stock with the serial number 1111111 is deleted. 
+        Warenager will tell user that serial number `33333333` does not map to any stock.
```

Comment:
Should this line follow the format of the others such as what is shown in "status message"?